loading.bs

```diff
@@ -818,13 +831,23 @@ A [=request=] |browserRequest| <dfn>matches the stored request</dfn>
 1. If |browserRequest|'s [=request/url=] is not [=url/equal=] to
     |storedRequest|'s [=request/url=], return "mismatch".
 
-1. Issue: Flesh this out, along the lines of
-    https://docs.google.com/document/d/1JdJkdY7cK2rD9JXbK75sSn069c0rhF9UnLQACBZ1OtY/edit.
-
+1. [=list/For each=] header (|name|, |storedValue|) in |storedRequest|'s
+    [=request/header list=]:
+    1. If |name| doesn't name a [=content negotiation mechanism specified to be
+        usable with Variants=], return "mismatch".
+    1. Otherwise, let |variantMatches| be the algorithm for selecting a result
+        defined by |name|'s content negotiation mechanism.
+    1. If |browserRequest|'s [=request/header list=] [=header list/does not
+        contain=] |name|, return "mismatch".
```

Comment:
I think this is wrong, but I may be misreading this. I would think that if the cached response is `Accept-Language: foo` and the browser doesn't specify an `Accept-Language`, this would match, per the [specified content-negotiation mechanism](https://httpwg.org/http-extensions/draft-ietf-httpbis-variants.html#content-language).

Reply:
It was out of date, now that Fetch has removed "combined value".

Variants doesn't seem to handle the case where a request header is just missing: https://github.com/httpwg/http-extensions/issues/731.

Reply:
I think support for null values of negotiable request headers is a necessity, though. Otherwise, if 3 different UAs support 3 different sets of content negotiation algorithms, then it would be impossible to generate an SXG that could be used by all of them. The union of all supported headers would fail because of the first item above, and the intersection of all supported headers would fail because of this item.

Even given a single UA, it would make it difficult, as one would have to tailor the header list to the particular version of the UA.

(I hope I'm not misunderstanding; I haven't spent a lot of time thinking about this, so I may be.)

Reply:
I agree. Now I just delegate to the content-negotiation algorithms, which I'm fixing in https://github.com/httpwg/http-extensions/pull/735.